server.go

```diff
@@ -324,7 +318,7 @@ func (s *server) channelHandleWrapper(handler ChannelHandler, handlerFunc Channe
 		}
 
 		// if no events were created we still want to log this to the channel, do so
-		if len(events) == 0 {
+		if len(events) == 0 && channel != nil {
```

Comment:
in what scenario do we get here without a channel but with events?

Reply:
we use the URL `/c/fba/receive` for the channel type so we will figure out the channel in the handler(not here by UUID)

Reply:
Why isn't the channel being returned by the FBA channel in `GetChannel`?

Reply:
Right that is not needed

Reply:
I remembered why we needed this we only add channel logs when we have a channel matched
https://github.com/nyaruka/courier/pull/311